biblissima-data-manifest-v1.json

```diff
@@ -4,7 +4,8 @@
         "name": "Biblissima Authority Files",
         "root": "https://data.biblissima.fr/w/",
         "main_page": "https://data.biblissima.fr/w/Accueil",
-        "api": "https://data.biblissima.fr/w/api.php"
+        "api": "https://data.biblissima.fr/w/api.php",
+        "mandatoryMediaInfoPropertyIds": "P7482, P170, P6216"
```

Comment:
I don't think we should provide `mandatoryMediaINfoPropertyIds` if the Wikibase instance doesn't use MediaInfo entities. Especially since the property identifiers you added here probably don't make sense on that Wikibase instance.
So I think I would only add it to the Wikimedia Commons manifest. It should also use an array syntax.